src/install/oracle/whatsNext.mdx

```diff
@@ -0,0 +1,7615 @@
+---
+componentType: default
+headingText: Metrics collected by the integration
+---
+## Oracle DB instance settings [#instance-settings]
```

Comment:
This doesn't appear in the right nav, which I definitely think it's worth troubleshooting why. This is the reference part of the material that people would come back to.